roomacoustics/sti.py

```diff
@@ -0,0 +1,176 @@
+"""
+Returns a np array with the female or male STI (speech transmission index), a 
+single number value on a metric scale between 0 (bad) and 1 (excellent) for 
+quality assessment of speech transmission channels.
+
+The indices are based on the modulation transfer function (MTF) that determinines
+affections of the intensity envelope throughout the transmission. The MTF values
+are assessed from the IR and are further modified based on auditory, ambient 
+noise and masking aspects.
+
+STI considers 7 octaves between 125 Hz and 8 kHz (125 Hz is not considered for
+the female STI) and 14 modulation frequencies between 0.63 Hz and 12 Hz. 
+
+
+============================
+
+Reference
+---------
+..  [1] IEC 60268-16:2011
+    Sound system equipment - Part 16: Objective rating of speech intelligibility 
+    by speech transmission index
+
+"""
+
+import warnings
+from pyfar import Signal  
+import pyfar.dsp.filter as filt
+import numpy as np
+
+def sti(signal, data_type=None, gender='male', level=np.array([np.nan]), 
+            snr=np.array([np.nan])):
+    """
+    Parameters
+    ---------
+    signal : Signal
+        The IR to be analyzed. Length must be at least 1.6 s and not shorter than
+        1/2 RT60. [1],  p. ??
+
+    data_type : 'electrical', 'acoustical'
+        Determines weather input signals are obtained acoustically or electrically.
+        Auditory effects can only be considered when "acoustical" [1], p. ??. 
+        Default is 'none'.
+
+    gender: 'female', 'male'
+        Defines the applied weighting factors. Default is 'male' as the STI 
+        algorithm is initially based on the male speech spectrum.
+
+    noise: np.array, 0
+        Level of the test signal in octave for bands 125 Hz - 8000 Hz in dB_SPL.
+
+    snr: np.array, 0
+        SNR in octave for bands 125 Hz - 8000 Hz in dB.
```

Comment:
- You could mention the shape of the array for an easier understanding.
- There should be a better explanation for how the SNR is defined or a link to the Section in the DIN where it is defined :)